### PR TITLE
Permanent Cookie

### LINKED ISSUE #10: Device Cookie Lasts Forever
As a device I want my cookie to last forever if possible so that things “just work” when I switch on.

Cookie currently being set in DevicesController#create

### PR DESCRIPTION
**Pull request by ghost**
Enables devices to have a permanent cookie so that the program can remember them even after the close their browser.

closes#10

### CODE REVIEW

**Comment by ghost** [comment]
@s-espinosa check it out!

File: spec/features/cookie_stays_permanent_after_closing_window_spec.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+RSpec.describe "Cookie stays permanent" do
+  scenario "device's cookie last 'forever' after closing window" do
+    device = FactoryGirl.create(:device)
+
+    visit '/'
+
+    time = get_me_the_cookies.first[:expires].strftime("%B %d, %Y")
+
+    expect(time).to eq("June 08, 2036")
```
Comment (s-espinosa) [inline spec/features/cookie_stays_permanent_after_closing_window_spec.rb]: Won't this fail tomorrow? Think we should figure out how to calculate this date.

**Comment by ghost** [comment]
@s-espinosa: hmmm, should I just test the year?

File: spec/factories/device.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
+  factory :device do
+    device_code "12345"
+  end
```
Comment (s-espinosa) [inline spec/factories/device.rb]: Have a similar implementation in my admin branch. This also looks good. Let's discuss tomorrow and figure out how we want to merge.

File: spec/features/cookie_stays_permanent_after_closing_window_spec.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe "Cookie stays permanent" do
+  scenario "device's cookie last 'forever' after closing window" do
+    device = FactoryGirl.create(:device)
```
Comment (s-espinosa) [inline spec/features/cookie_stays_permanent_after_closing_window_spec.rb]: I added the FactoryGirl::Syntax::Methods to a spec/support/factory_girl file. Then we can just say 'create(:device)' I think either way is fine, but leaving a note to remind myself to remove that if we decide to go this way. Let's discuss.

**Pull request merged**